### ISSUE TITLE
New Features I Want

### ISSUE THREAD

**Issue by stevencohn** [stevencohn/OneMore#39]
Here's a list of features I would like to see in OneMore, in no particular order...

* Add/remove numbering to pages and sections. This would allow numeric or alphabetic numbering on sections and either "1, 1.1, 1.1.1" or "1, a, i, ..." style numbering for hierarchical pages in a section
* Add diagnostic information to About box including special folders with links to open them in file explorer. This information is now available in the log file when you press F8 but adding it to the About box makes it more accessible to most users.
* Enhance Find/Replace to use the built-in Find API which would select the next match and highlight all subsequent matches.
* Add Auto-text translations where you enter short names or abbreviations and click an "auto-expand" button to replace them with text or content snippets
* Crop/Rotate images
* Hide/Show the page title date and time fields
* Clean/remove all author information from the page; not sure why this is needed but many users seem to like it?!
* Insert phonetic ruby text after selected text. This should work for the current system language.

**Comment by piyush1908** [comment]
Export as html (can you please allow entire section)
Can you also allow export as very long continuous pdf (I wonder)

**Comment by stevencohn** [comment]
I'll consider HTML export of a section.
OneNote already lets you export an entire section as a single PDF

**Comment by piyush1908** [comment]
I mean single continuous pdf (entire section on one page like it appears on
html)

**Comment by stevencohn** [comment]
Yes OneNote already has that built in. Go to File, export, choose Section, and export as pdf
